setup.py

```diff
@@ -27,6 +27,7 @@
     "ipywidgets",
     # notebook database example
     "jupysql",
+    "altair",
```

Comment:
please add a comment explainign that altair is needed for interactive confusion matrix